### ISSUE TITLE
[READ THIS!] This repository is not being maintained anymore. Use nicolas-chaulet/openapi-typescript-codegen instead!

### ISSUE THREAD

**Issue by dagadbm** [ferdikoomen/openapi-typescript-codegen#2064]
https://github.com/nicolas-chaulet/openapi-typescript-codegen

**Comment by dagadbm** [comment]
yes I just used the latest version of the fork to give me the option to use custom `BASE`

**Comment by ferdikoomen** [comment]
@dagadbm @nicolas-chaulet

**Comment by ferdikoomen** [comment]
@dagadbm This repository will hopefully be moved soon with @mrlubos his branch as base. I will need to work out the details with him, but things will be updated shortly.

**Comment by dagadbm** [comment]
i noticed that `@hey-api/openapi-ts` has been created. is this the new place?

**Comment by tobiashochguertel** [comment]
I don't like the Readme.md file of the fork - it looks so complicated, and I don't know where to start. I like the origin Readme.md file, it has only the necessary information and I get quickly started. The fork should show a similar Readme with a headline about the differences between the origin and the fork. Also, a headline about migration would be interesting. Everything else could be stored in a wiki page or another Markdown document in the repository.

I also think that this repository is still active, and the fork should be closed and integrated into this repository and add the developers of the fork as maintainers to this repository.

**Comment by jordanshatford** [comment]
@tobiashochguertel  Hey, unfortunately this repository is not being actively maintained, see [here](https://github.com/ferdikoomen/openapi-typescript-codegen/issues/2064#issuecomment-2002593442) and [here](https://github.com/ferdikoomen/openapi-typescript-codegen/issues/1276#issuecomment-1302392146). I do understand that changes were recently merged, but the code is not being actively maintianed.

Because of this we decided to make a fork (under an organization account) to prevent this from happening again. The reasoning was that with the code under an organization, many maintainers could more easily help without relying on an owner to oversee everything.

We do understand that the README in the fork is quite complicated, and we do plan to provide better documentation (via a webpage rather than GitHub Wiki). We have been focusing our time on fixing issues, cleaning up code, and adding additional features to the fork. Because of this we have not had time to work on documentation. 

If you have any recommendations for how we can better the documentation please open a ticket in the repository, or submit a PR to address it. Thank you, and hopefully we can provide better documentation in the near future.

**Comment by mrlubos** [comment]
@tobiashochguertel if you're familiar with this package, our fork isn't too dissimilar. The quick start section has a single command which is all you need to get started https://github.com/hey-api/openapi-ts?tab=readme-ov-file#quick-start
